### PR TITLE
Add method to get elapsed time

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Add method to get elapsed time
Related to https://github.com/acompany-develop/QuickMPC-libClient-py/pull/14
# Purpose
To be able to get the elapsed time from the client
# Contents
gRPC protobuf update
Add method to get elapsed time
# Testing Methods Performed
`make test t=./ManageContainer`
`make test t=./LibClient p=medium`

### CODE REVIEW

File: src/ManageContainer/Server/LibcToManageContainer/L2m.go [hunk 1]
```diff
@@ -384,6 +384,33 @@ func (s *server) GetDataList(ctx context.Context, in *pb.GetDataListRequest) (*p
...
+	AppLogger.Info("Get Elapsed time;")
+	AppLogger.Info("jobUUID: " + in.GetJobUuid())
+	JobUUID := in.GetJobUuid()
+	token := in.GetToken()
+
+	errToken := s.authorize(token, []string{"demo", "dep"})
+	if errToken != nil {
+		return &pb.GetElapsedTimeResponse{
+			IsOk: false,
+		}, errToken
```
[the remark this answers is not in the document]
Comment (arukuka) [reply src/ManageContainer/Server/LibcToManageContainer/L2m.go:397]: optional

**Review by arukuka — CHANGES_REQUESTED** [review]

File: src/Proto/LibcToManageContainer/libc_to_manage.proto [hunk 2]
```diff
@@ -167,3 +168,13 @@ message GetDataListResponse {
...
 }
+
+message GetElapsedTimeRequest {
+    string job_uuid = 1;
+    string token = 2;
+}
+
+message GetElapsedTimeResponse {
+    bool is_ok = 1;
+    string result = 2;
```
Comment (arukuka) [inline src/Proto/LibcToManageContainer/libc_to_manage.proto:179]:
```suggestion
    double elapsed_time = 2;
```

**Review by arukuka — CHANGES_REQUESTED** [review]

**Comment by KotaTakahashi9320** [comment]
@arukuka @come2ry @mdonaka 
諸々修正しましたので、再度ご確認お願いいたいします。

File: src/ManageContainer/Client/ManageToDb/Client_test.go [hunk 3]
```diff
@@ -342,5 +343,29 @@ func TestInsertModelParamsParallelRejectDuplicate(t *testing.T) {
...
+	expect := 98.302
+
+	client := Client{}
+	elapsedTime, err := client.GetElapsedTime(defaultJobUUID)
+
+	if err != nil {
+		t.Error("get elapsed time failed: " + err.Error())
+	}
+	if math.Abs(elapsedTime - expect) > 0.001 {
+		t.Errorf("get elapsed time failed: elapsed_time must be %f, but value is %f.",expect,elapsedTime)
```
Comment (mdonaka) [inline src/ManageContainer/Client/ManageToDb/Client_test.go:366]: typo: elapsed_time -> elapsedTime
```suggestion
		t.Errorf("get elapsed time failed: elapsedTime must be %f, but value is %f.",expect,elapsedTime)
```

Comment (KotaTakahashi9320) [reply src/ManageContainer/Client/ManageToDb/Client_test.go]: @mdonaka 
Thank you for your review
Please check the correction.

**Review by mdonaka — APPROVED** [review]

**Review by arukuka — APPROVED** [review]
Thank you!

**Pull request merged**